package.json

```diff
@@ -40,9 +40,9 @@
     }
   ],
   "scripts": {
-    "build": "tsc --module commonjs --declaration applicationinsights.ts",
-    "prepublish": "tsc --module commonjs --declaration applicationinsights.ts",
-    "pretest": "find Tests -type f -name \"*.ts\" | xargs tsc --module commonjs",
+    "build": "tsc --module commonjs --target es5 --declaration applicationinsights.ts",
```

Comment:
According to CI the only problem is with use of Object.keys() deep in a helper method. I'll check that out, but it seems to indicate the rest is okay?